examples/desktop/at_talk/src/main.c

```diff
@@ -235,7 +235,7 @@ static void *monitor_handler(void *xargs) {
   const char *from_atsign = args->from_atsign;
 
   pthread_mutex_lock(&monitor_mutex);
-  if ((ret = atclient_monitor_pkam_authenticate(monitor, atserver_host, atserver_port, atkeys, from_atsign)) != 0) {
+  if ((ret = atclient_monitor_pkam_authenticate(monitor, from_atsign, atkeys, NULL)) != 0) {
```

Comment:
This signature is changed, host and port are now part of the options struct (which can be NULL, and the pkam_authenticate will look it up for you)